contents/blog/company-survey-h1-2024.md

```diff
@@ -0,0 +1,99 @@
+
+We had a 79% response rate, or 33 out of 42 eligible respondents. For our last survey, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.
+
+We asked 14 questions based on the ones used by [Culture Amp](https://www.cultureamp.com/). Questions cover categories such as company, confidence, culture, and growth. 12 of the questions we asked required a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree:
+
+* PostHog is in a position to really succeed over the next three years.
+* I understand PostHog's goals and can see how my work contributes to them.
+* At PostHog, we have open and honest two-way communication.
+* I receive appropriate recognition for my work at PostHog.
+* I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies.
+* My manager and team around me genuinely care about my wellbeing.
+* The leaders at PostHog keep people informed about what is happening.
+* I feel like I am learning and growing at PostHog.
+* Generally, I believe my workload is reasonable for my role and I am able to arrange time out from work when I need it.
+* The support I am receiving and processes we have in place allow me to do my best possible work.
+* I would recommend PostHog as a great place to work.
+* I see myself still working at PostHog in two years' time.
+
+And two of which were open text fields:
+
+* What motivates you right now?
+* If you were to leave PostHog, what would the reason be?
+
+We took the scores we received and compared them to Culture Amp’s benchmark for ‘new tech’ companies with less than 200 people. We then took the average score out of 5 and multiplied it by 20 to get a percentage. It’s a bit rough, but is useful for comparing our scores against the industry benchmark.
+
+We ran this survey using the Pulse Surveys app in Slack. Our previous three surveys were quarterly, but this time we waited six months. We’ll stick with bi-annual surveys moving forward.
+
+### Highest scores
+
+* **97% positive:** I understand PostHog’s goals and can see how my work contributes to them.
+* **97% positive:** PostHog is in a position to really succeed over the next three years.
+* **97% positive:** My manager and team around me genuinely care about my wellbeing. 
+
+### Lowest scores
+
+* **82% positive:** The support I am receiving and processes we have in place allow me to do my best possible work.
+* **79% positive:** I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies. 
+* **79% positive:** I see myself still working at PostHog in two years' time. 
+
+### Qualitative questions
```

Comment:
'Fun' fact - we don't use AI to summarize this at our scale, we think it's worth a human reading and synthesizing it properly